bookkeeper-server/pom.xml

```diff
@@ -178,8 +208,26 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-all</artifactId>
+      <version>${netty.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-tcnative-boringssl-static</artifactId>
+      <version>${netty-tcnative.version}</version>
+      <classifier>${os.detected.classifier}</classifier>
```

Comment:
@revans2  @sijie @kishorekasi I think it is better to drop the  linux-x86_64 classifier, so that we will import only the pura Java code without any arch specific dependency